pod/perldiag.pod

```diff
@@ -3886,6 +3886,30 @@ mutable before freeing the ops.
 
 (F) You don't have System V message IPC on your system.
 
+=item Multidimensional hash lookup is disabled, is this code correct?
+
+(S syntax) You supplied a list of subscripts to a hash lookup under
+C<< no feature "multidimensional"; >>, eg:
+
+  $z = $foo{$x, $y};
+
+which by default acts like:
+
+  $z = $foo{join($;, $x, $y)};
+
+Under C<< no feature "multidimensional"; >> this replacement isn't
+done and the code behaves like:
+
+  $z = $foo{($x, $y)[-1]}; # last value of the list is the index
+
+This has no effect on code like:
```

Comment:
This could be clarified what 'this' refers to, namely the multidimensional "feature" itself, not the disabling of it.